plc4j/drivers/firmata/src/test/java/org/apache/plc4x/java/firmata/readwrite/FirmataManualTest.java

```diff
@@ -30,7 +30,7 @@ public static void main(String[] args) throws Exception {
 
         // Default for the Firmata Arduino sketch is, 57600 baud, 8 data-bits, no parity and one stop bit (Which is the default for the Serial transport)
         try(PlcConnection connection = PlcDriverManager.getDefault().getConnectionManager().getConnection("firmata:///dev/tty.usbmodem2114401")) {
-            PlcSubscriptionRequest subscriptionRequest = connection.subscriptionRequestBuilder().addEventTagAddress("analog1", "analog:1").addPreRegisteredConsumer("analog1", plcSubscriptionEvent -> {
+            PlcSubscriptionRequest subscriptionRequest = connection.subscriptionRequestBuilder().addEventTagAddress("analog1", "analog:1").setConsumer(plcSubscriptionEvent -> {
```

Comment:
shouldn't the consumer be set then as 3rd argument in `addEventTagAddress` following that logic? otherwise that would semantical not the same

Reply:
That was an example of how to register a handler for all tags of the request ... in that case it was only one, so yes ... there would be no difference to using the handler registation for an individual tag.